packages/cli/package.json

```diff
@@ -51,6 +50,7 @@
     "oclif": "^3",
     "shx": "^0.3.3",
```

Comment:
Does the CLI need to have depednecy on the plugin for it to work?
(assuming our first plugins would be pre-bundled)

Reply:
If we want the user to not have to install the plugin, I guess we can specify it as an explicit dep.

BTW: if the plugin is not specified in the deps, not sure how it would work with npx.

Reply:
ok, added to the deps.

still need to solve the case of npx + unbundled plugins

Reply:
Why wouldn't it work with npx?

Reply:
I guess it can always work with `-p`:
```bash
npx -p preevy -p @bla/my-preevy-plugin ...
```